components/Modal/Schedule/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+import dayjs from "dayjs";
+import { useShallow } from "zustand/react/shallow";
+
+import Input from "@/components/Input";
+import { useDateStore } from "@/store/useDateStore";
+import { useModalStore } from "@/store/useModalStore";
+
+import styles from "./ScheduleModal.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ScheduleModalProps {
+  mode: "create" | "edit" | "detail";
+}
+
+export default function ScheduleModal({ mode }: ScheduleModalProps) {
+  const [isReadOnly, setIsReadOnly] = useState<boolean>(mode === "detail");
+  const { closeModal } = useModalStore();
+  const { scheduleStart, scheduleEnd, setScheduleStart, setScheduleEnd } = useDateStore(
+    useShallow((state) => ({
+      scheduleStart: state.scheduleStart,
+      scheduleEnd: state.scheduleEnd,
+      setScheduleStart: state.setScheduleStart,
+      setScheduleEnd: state.setScheduleEnd,
+    })),
+  );
+
+  const handleCloseModal = () => {
+    closeModal();
+    setScheduleStart(null);
+    setScheduleEnd(null);
+  };
+
+  const handleSubmit = (mode: "create" | "edit") => {
+    //create 이면 post, edit이면 put 요청 추가
+    handleCloseModal();
+  };
+
+  return (
+    <div className={cn("container")}>
+      <form className={cn("create-schedule-form")}>
+        <input
+          type="text"
+          className={cn("schedule-name")}
+          readOnly={isReadOnly}
+          placeholder="무슨 일정인가요?"
+        />
+      </form>
```

Comment:
```suggestion
      <form className={cn("create-schedule-form")}>
        <input
          type="text"
          className={cn("schedule-name")}
          readOnly={isReadOnly}
          placeholder="무슨 일정인가요?"
        />
      </form>
```
P4: 지금 봤는데 `form`태그가 아래의 `input`태그는 안감싸고 여기 일정 이름만 감싸는 이유가 있으신가요??

Reply:
헉실수입니다! 감사합니다!